File: Sumin/MaskInfo_java/app/src/main/java/org/techtown/maskinfo_java/repository/MaskService.java
package org.techtown.maskinfo_java.repository;

import org.techtown.maskinfo_java.model.StoreInfo;

import retrofit2.Call;
import retrofit2.http.GET;

// https://gist.githubusercontent.com/junsuk5/bb7485d5f70974deee920b8f0cd1e2f0/raw/063f64d9b343120c2cb01a6555cf9b38761b1d94
public interface MaskService {
    String BASE_URL = "https://gist.githubusercontent.com/junsuk5/bb7485d5f70974deee920b8f0cd1e2f0/";

    @GET("raw/")
    Call<StoreInfo> fetchStoreInfo();
}

File: mask_kotlin/app/src/main/java/org/techtown/mask_kotlin/repository/MaskService.kt
package org.techtown.mask_kotlin.repository

import org.techtown.mask_kotlin.model.StoreInfo
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MaskService {
    @GET("raw/?m=5000")
    suspend fun fetchStoreInfo(
        @Query("lat") lat: Double,
        @Query("lng") lng: Double
    ): StoreInfo

    companion object {
        const val BASE_URL =
            "https://gist.githubusercontent.com/junsuk5/bb7485d5f70974deee920b8f0cd1e2f0/"
    }
}